rpm/SPECS/pepProxy.spec

```
@@ -170,7 +170,24 @@ rm -rf $RPM_BUILD_ROOT
 %{_install_dir}
 
 %changelog
-* Thu Mar 24 2021 Alvaro Vega <alvaro.vegagarcia.com> 1.14.0
+* Mon May 23 2022 Alvaro Vega <alvaro.vegagarcia.com> 1.15.0
+- Add INSPECT_ENABLED env var to enable node inspection/debuging (#489)
+- Add: conf and env var (DISABLE_DOMAIN_MIDDLEWARE) to disable domain middleware to reduce overhead (but loosing some info in logs) (#498)
+- Add: new API to retrieve and reset cache stats (GET, DELETE /admin/cacheStats)
+- Add: new API to reset cache (DELETE /admin/cache)
+- Remove: unrequired dep underscore
+- Fix: Dockerfile to include initial packages upgrade
```

Some minor cleanup:

```suggestion
- Add: INSPECT_ENABLED env var to enable node inspection/debuging (#489)
- Add: conf and env var (DISABLE_DOMAIN_MIDDLEWARE) to disable domain middleware to reduce overhead (but loosing some info in logs) (#498)
- Add: new API to retrieve and reset cache stats (GET, DELETE /admin/cacheStats)
- Add: new API to reset cache (DELETE /admin/cache)
- Fix: Dockerfile to include initial packages upgrade
- Remove: unrequired dep underscore
```

In addition, CNR file should be flushed in this PR.